bindings/nodejs/tests/fixtures/random

```diff
@@ -0,0 +1,19 @@
+Enjoyed minutes related as at on on. Is fanny dried as often me. Goodness as reserved raptures to mistaken steepest oh screened he. Gravity he mr sixteen esteems. Mile home its new way with high told said. Finished no horrible blessing landlord dwelling dissuade if. Rent fond am he in on read. Anxious cordial demands settled entered in do to colonel.
```

Comment:
Do we have license issues for those content?

It's better to generate random files by scripts instead.